src/Drivers/KosevenTinkerwellDriver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+
+class KosevenTinkerwellDriver extends TinkerwellDriver
+{
+    /**
+     * @param string $projectPath
+     * @return bool
+     */
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/application/bootstrap.php');
```

Comment:
Is this the only way to detect the framework? This looks like it might not be specific enough

Reply:
Sorry for very very late response. Yup, I think that's the standard way to bootstrap the framework, it works for both [Koseven](https://github.com/koseven/koseven/blob/master/application/bootstrap.php) and deprecated [Kohana](https://github.com/kohana/kohana/blob/3.3/master/application/bootstrap.php). 

And I've been using Tinkerwell all the time via having this customer driver under `.tinkerwell` directly in the root directory of our legacy application, it works well, really enjoying Thinkerwell 😃

Reply:
Hi @mpociot, is there any chance this will get merged? otherwise I can close it if you think it's not worth to adding it.